src/constants/constants.js

```diff
@@ -1 +1 @@
-export const REACT_APP_SERVER_URL="http://127.0.0.1:8080";
\ No newline at end of file
+export const REACT_APP_SERVER_URL="http://127.0.0.1:8000";
```

Comment:
Can you pull from `main` branch to fetch some of the changes that @fi5421 did for environment variables